eth_defi/deploy.py

```diff
@@ -32,6 +31,7 @@ def deploy_contract(
     deployer: str,
     *constructor_args,
     register_for_tracing=True,
+    proxy_address: HexAddress = None,
```

Comment:
Argument docstring missing "why"

Reply:
This wasn't used, I reverted this change